internal/operators/ocs/ocs_operator_test.go

```diff
@@ -0,0 +1,96 @@
+package ocs_test
+
+import (
+	"context"
+	"fmt"
+
+	. "github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/extensions/table"
+	. "github.com/onsi/gomega"
+	"github.com/openshift/assisted-service/internal/common"
+	"github.com/openshift/assisted-service/internal/operators/ocs"
+	"github.com/openshift/assisted-service/models"
+)
+
+var _ = Describe("Ocs Operator Test GetHostRequirements", func() {
+	var (
+		ctx            = context.TODO()
+		operator       = ocs.NewOcsOperator(common.GetTestLog())
+		master1        = &models.Host{Role: models.HostRoleMaster, Inventory: defaultInventory()}
+		master2        = &models.Host{Role: models.HostRoleMaster, Inventory: defaultInventory()}
+		master3        = &models.Host{Role: models.HostRoleMaster, Inventory: defaultInventory()}
+		worker1        = &models.Host{Role: models.HostRoleWorker, Inventory: ocsInventoryWithDisks(12, 64000000000)}
```

Comment:
what about `64*conversions.GB`. Adding something like this to the [conversions.go](https://github.com/openshift/assisted-service/blob/master/pkg/conversions/conversions.go) file:

```
const (
	KB = 1000
	MB = 1000 * KB
	GB = 1000 * MB
	TB = 1000 * GB
	PB = 1000 * TB
)
```